x-pack/plugins/reporting/public/management/mount_management_section.tsx

```diff
@@ -5,55 +5,54 @@
  * 2.0.
  */
 
-import { I18nProvider } from '@kbn/i18n-react';
-import * as React from 'react';
+import React, { Suspense, lazy } from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import { Observable } from 'rxjs';
-import { CoreSetup, CoreStart } from '@kbn/core/public';
-import { ILicense } from '@kbn/licensing-plugin/public';
-import { KibanaThemeProvider } from '@kbn/react-kibana-context-theme';
-import { ReportingAPIClient, InternalApiClientProvider } from '../lib/reporting_api_client';
-import { IlmPolicyStatusContextProvider } from '../lib/ilm_policy_status_context';
-import { ClientConfigType } from '../plugin';
+
+import { EuiLoadingSpinner } from '@elastic/eui';
+
+import type { CoreSetup, CoreStart } from '@kbn/core/public';
+import type { ILicense } from '@kbn/licensing-plugin/public';
+import { ErrorBoundary, ErrorBoundaryKibanaProvider } from '@kbn/shared-ux-error-boundary';
+
+import type { ReportingAPIClient } from '../lib/reporting_api_client';
+import type { ClientConfigType } from '../plugin';
 import type { ManagementAppMountParams, SharePluginSetup } from '../shared_imports';
-import { KibanaContextProvider } from '../shared_imports';
-import { ReportListing } from '.';
-
-export async function mountManagementSection(
-  coreSetup: CoreSetup,
-  coreStart: CoreStart,
-  license$: Observable<ILicense>,
-  config: ClientConfigType,
-  apiClient: ReportingAPIClient,
-  urlService: SharePluginSetup['url'],
-  params: ManagementAppMountParams
-) {
+
+const ManagementApp = lazy(() =>
+  // component is not the default export of the module
+  import('./management_app').then((module) => ({ default: module.ManagementApp }))
```

Comment:
This PR changes the way the main component for the Reporting Mgmt app is imported, to make it possible to wrap the component in the `ErrorBoundary`